flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -2416,10 +2416,17 @@ class ArrayExprLowering {
   };
 
   using ExtValue = fir::ExtendedValue;
-  using IterSpace = const IterationSpace &;      // active iteration space
-  using CC = std::function<ExtValue(IterSpace)>; // current continuation
-  using PC =
-      std::function<IterationSpace(IterSpace)>; // projection continuation
+  /// Active iteration space.
+  using IterSpace = const IterationSpace &;
+  /// Current continuation. Function that we generate IR for a single iteration
+  /// of the pending iterative loop structure.
+  using CC = std::function<ExtValue(IterSpace)>;
+  /// Projection continuation. Function that will project one iteration space
+  /// into another.
+  using PC = std::function<IterationSpace(IterSpace)>;
```

Comment:
nit: `IterSpace(IterSpace)` or `IterationSpace(IterationSpace)` for consistency.

Reply:
They are different types however.